Source/Core/Common/IniFile.cpp

```diff
@@ -133,6 +168,18 @@ bool IniFile::Section::Get(const std::string& key, int* value, int defaultValue)
   return false;
 }
 
+bool IniFile::Section::Get(const std::string& key, s64* value, s64 default_value) const
+{
+  std::string temp;
+  bool retval = Get(key, &temp);
+
+  if (retval && TryParse(temp, value))
```

Comment:
Also, ideally you would do:

`return Get(key, &temp) && TryParse(temp, value)`

and avoid the temporary in my last comment.  I think the order of operations should work but I'd test it to be sure!